app/src/main/java/com/example/cleanarchtest/CATApplication.kt

```diff
@@ -1,10 +1,7 @@
 package com.example.cleanarchtest
 
 import android.app.Application
-import com.example.cleanarchtest.di.ViewModelModule
-import com.example.cleanarchtest.di.NetworkModule
-import com.example.cleanarchtest.di.RepositoryModule
-import com.example.cleanarchtest.di.UseCaseModule
+import com.example.cleanarchtest.di.*
```

Comment:
Nit - We should avoid using wildcard imports (i.e. imports ending with `*`) as some lint checkers (like Ktlint) see them as errors and would cause our CI/CD pipeline(s) to fail.

Reply:
Here's how to disable wildcard imports in Android Studio/IntelliJ IDEA - https://stackoverflow.com/questions/67341957/how-do-i-disable-wildcard-imports-for-all-kotlin-files-in-android-studio